src/services/UserRequests.tsx

```diff
@@ -0,0 +1,52 @@
+export const signInRequest = async (body: object) => {
+  try {
+    const response = await fetch(
+      `${import.meta.env.VITE_BASE_URL}/users/sign_in`,
```

Comment:
`import.meta.env.VITE_BASE_URL` is repeated a lot. In order to have a cleaner code I think it will be better to define: `const BASE_URL = import.meta.env.VITE_BASE_URL ` and only use the const in all the file

Reply:
Love it! Fixed